composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "myparcelnl/sdk",
-    "version": "v3.1.2-beta.2",
+    "version": "v3.2.0-beta.1",
```

Comment:
Dit is een breaking change:
PICKUP_EXPRESS is een public constructor. Een programmeur kan PICKUP_EXPRESS gebruikt hebben in zijn code. Je moet dus v4.0.0 gebruiken of je moet
```
/**
 * @deprecated blabla
 */
```
boven `const PICKUP_EXPRESS = 'retailexpress';` zetten

Reply:
done